### PR TITLE
zboss: version 3.3.0.7+03_22_2021

### PR DESCRIPTION
**Pull request by maciekfabia**
Update production ZBOSS libraries to version 3.3.0.7+03_22_2021

### CODE REVIEW

File: zboss/production/include/osif/build_info.h [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2020 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```
Comment (tomchy) [inline zboss/production/include/osif/build_info.h:4]: Please fix the license tag (revert) and update date.

File: zboss/production/include/osif/libzboss_config.ed.h [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2020 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```
Comment (tomchy) [inline zboss/production/include/osif/libzboss_config.ed.h:4]: Please fix the license tag (revert) and update date.

File: zboss/production/include/osif/libzboss_config.h [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2020 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```
Comment (tomchy) [inline zboss/production/include/osif/libzboss_config.h:4]: Please fix the license tag (revert) and update date.

File: zboss/production/include/osif/zb_osif_platform.h [hunk 4]
```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2020 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```
Comment (tomchy) [inline zboss/production/include/osif/zb_osif_platform.h:4]: Please fix the license tag (revert) and update date.

File: zboss/production/include/osif/zb_macll.h [hunk 5]
```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2020 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```
Comment (tomchy) [inline zboss/production/include/osif/zb_macll.h:4]: Please fix the license tag (revert) and update date.

File: zboss/production/include/osif/mac_platform.h [hunk 6]
```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2020 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```
Comment (tomchy) [inline zboss/production/include/osif/mac_platform.h:4]: Please fix the license tag (revert) and update date.

File: zboss/production/include/osif/zb_config_platform.h [hunk 7]
```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2020 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```
Comment (tomchy) [inline zboss/production/include/osif/zb_config_platform.h:4]: Please fix the license tag (revert) and update date.

**Review by tomchy — CHANGES_REQUESTED** [review]

**Review by tomchy — APPROVED** [review]

File: zboss/production/include/osif/zb_config_platform.h (hunk 7 above)
Comment (maciekfabia) [reply zboss/production/include/osif/zb_config_platform.h]: License and copyright year fixed

**Review by rlubos — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**